data/adapters/ontologies_adapter.py

```diff
@@ -0,0 +1,128 @@
+from adapters import Adapter
+from owlready2 import *
+import rdflib
+
+
+class Ontology(Adapter):
+  UBERON = 'http://purl.obolibrary.org/obo/uberon.owl'
```

Comment:
Do we need to use the same ontologies used in IGVF Ontology: https://github.com/IGVF-DACC/ontology ?